UnitTests/project.json

```diff
@@ -26,10 +26,11 @@
           }
         }
     },
-    "dependencies": {
-        "jose-jwt": "2.0.0",
-        "xunit": "2.1.0"
-    },
+  "dependencies": {
+    "jose-jwt": "2.0.0",
+    "Microsoft.NETCore.App": "1.1.0",
```

Comment:
Hmmmm.... I'm sorry, I didn't know these files got changed. You should be able to ignore them if you wish. Really the only changes are in AesKeyWrap.cs and RsaUsingSha.cs